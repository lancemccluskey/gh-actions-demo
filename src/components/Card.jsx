/** @jsx jsx */
import { jsx } from 'theme-ui';

const Card = ({ children }) => {
  return (
    <div sx={{ variant: 'containers.card' }}>
      {children}
    </div>
  );
};

export default Card;
