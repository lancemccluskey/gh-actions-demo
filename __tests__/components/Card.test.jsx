import { render, screen } from '@testing-library/react';
import Card from '../../src/components/Card';

describe('<Card />', () => {
  it('renders', () => {
    const { container } = render(<Card />);

    expect(container).toMatchSnapshot();
  });
});
