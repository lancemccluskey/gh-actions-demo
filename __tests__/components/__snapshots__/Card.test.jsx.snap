// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> renders 1`] = `
<div>
  <div
    class="css-1nuvgh6-Card"
  />
</div>
`;
